templates/redis6-sentinel.conf.erb

```diff
@@ -0,0 +1,334 @@
+# *** IMPORTANT ***
```

Comment:
can you please convert the template to epp? We would like to get rid of erb on the long run and to achieve this, all new templates have to be epp.

Reply:
@bastelfreak Done